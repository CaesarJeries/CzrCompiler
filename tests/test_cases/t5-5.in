void main() {
    //if test

    if(true)
    {
        print("true\n");
    }

    if(false or false and  true)
    {
        print("true\n");
    }

    int ione;
    byte btwo;

    {
        ione = 3;
        btwo = 4b;
        if(ione < btwo)
            print("ione < btwo\n");
        if(ione > btwo)
            print("ione > btwo\n");
        else if(ione >= btwo  and  ione <= btwo )
            print("ione == btwo\n");
        else
            print("ione != btwo\n");
    }

    if (ione != btwo) {
        ione = 4;

        if(ione < btwo)
        {
            print("ione < btwo\n");
        }
        if(ione > btwo)
        {
            print("ione > btwo\n");
        }

        if(ione >= btwo  and  ione <= btwo )
        {
            print("ione == btwo\n");
        }
        else
        {
            print("ione != btwo\n");
        }
    }
}