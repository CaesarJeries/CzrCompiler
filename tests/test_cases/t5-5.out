Loaded: ./exceptions.s
true
ione < btwo
ione != btwo
ione == btwo
